siege_utilities/survey/render.py

```
@@ -0,0 +1,199 @@
+        fig = cg.create_choropleth_map(agg, geo_column=chain.geo_column, value_column="value")
+        return fig
+    except Exception:
+        return None
+
+
+# ---------------------------------------------------------------------------
+# Public API
+# ---------------------------------------------------------------------------
+
+def chain_to_argument(
+    chain: "Chain",
```

Ruff (0.15.10)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)